mods/default/nodes.lua

```diff
@@ -2034,9 +2036,9 @@ function default.register_chest(name, d)
 		end
 	})
 end
+default.register_chest = default.chest.register_chest -- Obsolete alias
 
```

Comment:
I forgot to leave this obsolescence note in the commit. Not sure how to document that.

Reply:
Why is this alias obsolete?

Reply:
Belongs into `legacy.lua`, same as an alias for the previous public `default.get_chest_formspec`.